addons/beehave/debug/graph_edit.gd

```diff
@@ -47,7 +47,7 @@ func _ready() -> void:
 	layout_button.flat = true
 	layout_button.focus_mode = Control.FOCUS_NONE
 	layout_button.pressed.connect(func(): horizontal_layout = not horizontal_layout)
-	get_zoom_hbox().add_child(layout_button)
+	get_box().add_child(layout_button)
```

Comment:
this will not work - you need to write the get_box() method yourself as suggested by me!

Reply:
Oh, sorry I only just started using Godot and didn't realize you were suggesting me to make that change haha

Where would I place such a function to make it accessible by multiple scripts? I'm used to C# where this could just be an extension method.

Reply:
Just define it in the graph_edit script